app/src/main/java/nodomain/freeyourgadget/gadgetbridge/activities/charts/LiveActivityFragment.java

```diff
@@ -432,7 +443,9 @@ private BarDataSet setupCommonChart(CustomBarChart chart, BarEntry entry, String
     }
 
     private BarDataSet setupTotalStepsChart(CustomBarChart chart, BarEntry entry, String label) {
-        mTotalStepsChart.getAxisLeft().setAxisMaximum(5000); // TODO: use daily goal - already reached steps
+        mTotalStepsChart.getAxisLeft().addLimitLine(new LimitLine(GBApplication.getPrefs().getInt(ActivityUser.PREF_USER_STEPS_GOAL, 10000), "ss")); // TODO: use daily goal - already reached steps
```

Comment:
We have a constant for the default value in ActivityUser.defaultUserStepsGoal. I see that there are other places where this needs to be fixed.